azure-mgmt-website/src/main/java/com/microsoft/azure/management/website/AppServicePlan.java

```diff
@@ -8,81 +8,46 @@
 
 import com.microsoft.azure.management.resources.fluentcore.arm.models.GroupableResource;
 import com.microsoft.azure.management.resources.fluentcore.arm.models.HasName;
+import com.microsoft.azure.management.resources.fluentcore.model.Appliable;
 import com.microsoft.azure.management.resources.fluentcore.model.Creatable;
 import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
 import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
 import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
 import com.microsoft.azure.management.website.implementation.AppServicePlanInner;
 
 /**
- * An immutable client-side representation of an Azure Web App.
+ * An immutable client-side representation of an Azure App Service Plan.
  */
 public interface AppServicePlan extends
         GroupableResource,
         HasName,
         Refreshable<AppServicePlan>,
         Updatable<AppServicePlan.Update>,
         Wrapper<AppServicePlanInner> {
-
-    /**
-     * @return target worker tier assigned to the App Service Plan.
-     */
-    String workerTierName();
-
-    /**
-     * @return app Service Plan Status. Possible values include: 'Ready', 'Pending'.
-     */
-    StatusOptions status();
-
-    /**
-     * @return app Service Plan Subscription.
-     */
-    String subscription();
```

Comment:
I deleted it because an app service plan cannot exist across subscriptions. The user already knows which subscription he is using - this is just noise exposed from web app API - if you keep reading you're going to see `resourceGroup()` as well: https://github.com/Azure/azure-sdk-for-java/pull/1221/files/99437701c7efe785475a13621f403af7d8fbae7e#diff-7f5e1fee787b1eb486ff55f7ab6d6dd7L80.